pkg/command/autoscaling/list.go

```diff
@@ -0,0 +1,158 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package autoscaling
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1beta1 "k8s.io/apimachinery/pkg/apis/meta/v1beta1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+	"knative.dev/client/pkg/kn/commands/flags"
+	hprinters "knative.dev/client/pkg/printers"
+	"knative.dev/kn-plugin-admin/pkg"
+	"knative.dev/serving/pkg/autoscaler/config"
+	"knative.dev/serving/pkg/autoscaler/config/autoscalerconfig"
+)
+
+// A function for getting specific field value from autoscaler config
+type valueOfConfig func(*autoscalerconfig.Config) string
+
+var (
```

Comment:
Should this be a constant? Since it should not change. Maybe difficult to do this with Golang... worth a try?

Reply:
Yes, I checked and found that Golang doesn't support declare map as constant. I think we need to keep using `var` if we want to still use map here.